testing/btest/btest.cfg

```diff
@@ -32,6 +32,8 @@ ZEEK_DISABLE_ZEEKYGEN=1
 ZEEK_ALLOW_INIT_ERRORS=1
 ZEEK_SUPERVISOR_NO_SIGKILL=1
 UBSAN_OPTIONS=print_stacktrace=1
+SPICY_PATH=%(testbase)s/../../auxil/spicy/spicy/spicy/lib:%(testbase)s/../../auxil/spicy/spicy/hilti/lib
+HILTI_CXX_INCLUDE_DIRS=%(testbase)s/../../auxil/spicy/spicy/hilti/runtime/include:%(testbase)s/../../auxil/spicy/spicy/spicy/runtime/include:%(testbase)s/../../build/auxil/spicy/spicy/include:%(testbase)s/../../build:%(testbase)s/../../build/src:%(testbase)s/../../build/src/include:%(testbase)s/../../src:%(testbase)s/../../auxil/broker/include
```

Comment:
Any chance to do without these? That seems not only fragile but may also hide problems. Generally, the Spicy tools should be able to figure this out on their own (though maybe that doesn't work for the Zeek-internal build)?

Reply:
See above.